core/trino-main/src/main/java/io/trino/metadata/Metadata.java

```diff
@@ -716,4 +717,9 @@ default boolean isMaterializedView(Session session, QualifiedObjectName viewName
      * Returns a table handle for the specified table name with a specified version
      */
     Optional<TableHandle> getTableHandle(Session session, QualifiedObjectName tableName, Optional<TableVersion> startVersion, Optional<TableVersion> endVersion);
+
+    /**
+        Returns maximal number of tasks that should be in writing to specific connector.
```

Comment:
`Returns maximum number of tasks that should be created while writing data to specific connector.`